pkg/api/convertertoagentpoolonlyapi.go

```diff
@@ -264,3 +266,63 @@ func convertVLabsAgentPoolOnlyCertificateProfile(vlabs *vlabs.CertificateProfile
 	api.KubeConfigCertificate = vlabs.KubeConfigCertificate
 	api.KubeConfigPrivateKey = vlabs.KubeConfigPrivateKey
 }
+
+func isAgentPoolOnlyCluster(version string, contents []byte) bool {
```

Comment:
Please remove this function. This is not the right way to check if it's an agent pool only cluster or not.